{"uid":"1526333ceaa0a6cf","name":"test_full_cluster_restart[SIGKILL-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"2702afd99304f2d258e13b03718c72a0","time":{"start":1714442277246,"stop":1714442344639,"duration":67393},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"failed","statusMessage":"AssertionError: Not all units down at the same time.\nassert False","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 275, in test_full_cluster_restart\n    assert await are_all_db_processes_down(\nAssertionError: Not all units down at the same time.\nassert False","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1714440514811,"stop":1714440514811,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1714440514812,"stop":1714440514812,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1714440514812,"stop":1714440515128,"duration":316},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1714442268807,"stop":1714442268807,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1714442268808,"stop":1714442268808,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition","time":{"start":1714442268808,"stop":1714442276169,"duration":7361},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"loop_wait","time":{"start":1714442276170,"stop":1714442277245,"duration":1075},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"loop_wait::finalizer","time":{"start":1714442344661,"stop":1714442344712,"duration":51},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition::finalizer","time":{"start":1714442344714,"stop":1714442352435,"duration":7721},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1714442352435,"stop":1714442358507,"duration":6072},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1714442358508,"stop":1714442358508,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1714443366494,"stop":1714443367062,"duration":568},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1714443367064,"stop":1714443367064,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1714443367064,"stop":1714443367064,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1714443367064,"stop":1714443367065,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-411-aa6309f95810e4df779039aa"},{"name":"thread","value":"9405-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGKILL'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":7,"broken":1,"skipped":0,"passed":9,"unknown":0,"total":17},"items":[{"uid":"964bab08481f2bfb","reportUrl":"../23//#testresult/964bab08481f2bfb","status":"passed","time":{"start":1714355895617,"stop":1714355928023,"duration":32406}},{"uid":"4ac636505e4f8608","reportUrl":"../22//#testresult/4ac636505e4f8608","status":"passed","time":{"start":1714269897846,"stop":1714269935454,"duration":37608}},{"uid":"88bf94e2af12c312","reportUrl":"../20//#testresult/88bf94e2af12c312","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7c555bea1b40 state=finished raised RetryError>]","time":{"start":1714184222984,"stop":1714184222984,"duration":0}},{"uid":"e53521931a03b4fc","reportUrl":"../19//#testresult/e53521931a03b4fc","status":"passed","time":{"start":1714098756240,"stop":1714098790898,"duration":34658}},{"uid":"357e5633f9c59656","reportUrl":"../18//#testresult/357e5633f9c59656","status":"passed","time":{"start":1714011718555,"stop":1714011754422,"duration":35867}},{"uid":"6076c3704def4afe","reportUrl":"../17//#testresult/6076c3704def4afe","status":"passed","time":{"start":1713925379898,"stop":1713925415715,"duration":35817}},{"uid":"4b56777c3eb6eadf","reportUrl":"../16//#testresult/4b56777c3eb6eadf","status":"passed","time":{"start":1713837473109,"stop":1713837501884,"duration":28775}},{"uid":"84f3b5e254dff9d4","reportUrl":"../15//#testresult/84f3b5e254dff9d4","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1713751138039,"stop":1713751205955,"duration":67916}},{"uid":"a148fb57d01303f1","reportUrl":"../11//#testresult/a148fb57d01303f1","status":"passed","time":{"start":1713491946977,"stop":1713491981347,"duration":34370}},{"uid":"ea58940351b78ddd","reportUrl":"../10//#testresult/ea58940351b78ddd","status":"passed","time":{"start":1713405399156,"stop":1713405433067,"duration":33911}},{"uid":"3c7eaa19a45f3265","reportUrl":"../9//#testresult/3c7eaa19a45f3265","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1713319566972,"stop":1713319868551,"duration":301579}},{"uid":"93978e3c03b0e501","reportUrl":"../8//#testresult/93978e3c03b0e501","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1713232822568,"stop":1713232889816,"duration":67248}},{"uid":"d5301699ea15f6d2","reportUrl":"../7//#testresult/d5301699ea15f6d2","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1713153244477,"stop":1713153311686,"duration":67209}},{"uid":"7a79982634afed9b","reportUrl":"../6//#testresult/7a79982634afed9b","status":"passed","time":{"start":1713060137028,"stop":1713060172962,"duration":35934}},{"uid":"8b8b12c161440949","reportUrl":"../4//#testresult/8b8b12c161440949","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1712973257889,"stop":1712973326361,"duration":68472}},{"uid":"6bcaba177a413a97","reportUrl":"../2//#testresult/6bcaba177a413a97","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1712887208860,"stop":1712887277689,"duration":68829}}]},"tags":["asyncio","group(1)"]},"source":"1526333ceaa0a6cf.json","parameterValues":["'postgres'","'SIGKILL'"]}